src/Buttons.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import VTMap from "./VTMap";
+import leafletPip from "@mapbox/leaflet-pip";
+import borderData from "./border";
+import L from "leaflet";
+import { Map } from "react-leaflet";
+
+class Buttons extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      disabled: true,
+      gameStarted: false,
+      long: "",
+      lat: "",
+      zoom: 7,
+      marker: {},
+      modalOpen: false,
+    };
+  }
+
+  //---------Handles click event-------------//
+  handleClick = (event) => {
+    this.setState({
+      gameStarted: true,
+    });
+  };
+
+  //---------Handles event after guess button is clicked-------------//
+  handleGuessButton = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      gameStarted: true
+    });
+  };
```

Comment:
You aren't using any of these functions. You are drawing all your on click methods from props which are defined on the parent component. Both the state, and the methods of this class are being ignored by your render function.